README.md

```diff
@@ -8,21 +8,38 @@
 [![license](https://img.shields.io/github/license/Skyscanner/backpack-android.svg)](https://github.com/Skyscanner/backpack-android)
 [![platform](https://img.shields.io/badge/platform-android-green.svg)](https://github.com/Skyscanner/backpack-android)
 
-## Installation
+Backpack for Android supports two targets: **Android View** system and **Jetpack Compose**.
 
-Backpack is available through [Maven Central](https://search.maven.org/artifact/net.skyscanner.backpack/backpack-android). To install
-all of it, add the following line to your `build.gradle` (in your app module) in the `dependencies` block:
+Backpack is available through Maven Central. Thus, before adding Backpack to your project,
+make sure Maven Central is in your repositories list. Add this to your root `build.gradle`:
 
 ```gradle
-implementation 'net.skyscanner.backpack:backpack-android:33.2.0'
+repositories {
+    mavenCentral()
+}
 ```
 
-If your app resolves dependencies through Maven Central you're all set, if not add in your root `build.gradle`
+### Demo application
+The Backpack demo application is a good way of referring to the variants available for a component and their correct usage.
+The code is available under `/app` directory.
+[The app can be downloaded from App Center](https://install.appcenter.ms/orgs/backpack/apps/backpack-android/distribution_groups/everyone)
+or by scanning the QR code below.
+
+![QR code](qr.png)
+
+## Android View system
+
+You can find the list of the available components, as well as the code samples and the screenshots [here](https://backpack.github.io/components/text?platform=android).
```

Comment:
Instead of having a duplicated list of the components (refering to Github without screenshots in place) it's better to redirect users to our website. The only issue is that we don't have (yet!) a link to all Android components, so I'm using most common one as a reference for now – with plans to support the URLs later